### ISSUE TITLE
Configure accordion so that the item div is not necessary

### ISSUE THREAD

**Issue by c2dev** [TheC2Group/accordion#1]
Is there a way to configure the accordion so that the item div isn't necessary? Primary reason being that in some cases it makes it impossible to make your code both valid and semantic. Like in the case of an FAQ list, which I believe should be marked up like so:

    <dl class="accordion">
    <dt class="target">Question 1</dt>
    <dd class="panel">Answer 1</dd>
    <dt class="target">Question 2</dt>
    <dd class="panel">Answer 2</dd>
    </dl>

**Comment by cuth** [comment]
Here is the problem:

Accordion is using the `item` container to animate the height attribute. This turned out to be the least *hacky* solution to measure the height of the `panel` without having to show and hide it really fast.

Solutions to measuring the height of the panel while it is `disabled` are welcome.

**Comment by mrself** [comment]
I do not understand why measuring height is important here. Why not use jQuery `slideDown`, `slideUp` functions which do the same?

**Comment by cuth** [comment]
Yes, jQuery makes animating height very easy. But the animation is not smooth and this was one of the main reasons to make this module.
